main.js

```diff
@@ -0,0 +1,140 @@
+/*Heather Sleyster
+sleyster_a02b
+INFO 2134
+Thoendel
+03/17/2020
+*/
+window.addEventListener('load', function() {
+/*These statements assign the Form element to the form variable 
+and the select element named 'meal' to the select variable.
+*/
+const form = document.getElementById('contactForm');
+const meal = document.getElementsByName('meal');
+
+/*This if statement checks the value of the selected meal and if
+it is empty removes the 'visible' class from the form. If breakfast
+is selected the visible class will be added to that option only, if
+lunch is selected the visible class will be added to that option only,
+if dinner is selected the visible class will be added to that option.
+*/
+
+if (meal.value == '') {
+    mealOption.classList.remove('hidden');
```

Comment:
what is mealOption...you will need to make a var like above - to grab this div by id. Then - if meal has a value (you are saying if it doesn't) I would think you would want to remove hidden and add visible.